src/main/java/org/isf/examination/gui/PatientExaminationEdit.java

```diff
@@ -1465,17 +1465,33 @@ public SwingActionSavePatientExamination() {
 		@Override
 		public void actionPerformed(ActionEvent actionEvent) {
 
-			try {
-				examinationBrowserManager.saveOrUpdate(patex);
-				modified = false;
-			} catch (OHServiceException ohServiceException) {
-				MessageDialog.showExceptions(ohServiceException);
+			if (patex.getPex_weight() != 0 && patex.getPex_height() != 0) {
```

Comment:
Instead of fetching `weight` and `height` potentially many times why not get them once and store them in a local variable?

Reply:
Thanks for the suggestion. This will  improve code and keep it clean.